src/app/pages/datasets/components/dataset-details-card/dataset-details-card.component.html

```diff
@@ -6,47 +6,47 @@ <h3 mat-card-title>{{ 'Dataset Details' | translate }}</h3>
       </mat-card-header>
       <mat-card-content>
         <div class="row">
-          <span class="header">Type: </span>
+          <span class="header">{{ 'Type: ' | translate }}</span>
```

Comment:
I believe semicolons and spaces are better to keep outside of the translatable string. More chances to be translated